app/services/weets.js

```diff
@@ -14,3 +14,12 @@ exports.findById = id =>
     logger.error('Error while trying to get weet by ID.', error.message);
     throw errors.databaseError(error.message);
   });
+
+exports.listWeets = async (page = 1, limit = 10) => {
```

Comment:
Imagine that you have a lot of request to get data. So, to avoid this default assignation, you can have a mapper file of pagination params like this:
```.js
const { DEFAULT_PAGE, DEFAULT_LIMIT } = require('../utils/constants');

exports.pagination = req => ({
  page: parseInt(req.query.page || DEFAULT_PAGE),
  limit: parseInt(req.query.limit || DEFAULT_LIMIT)
});

```

And call it when maps the parameters of request in controller files.